features/support/initial_data.rb

```diff
@@ -12,12 +12,8 @@
 
   ThreeScale.config.stubs(superdomain: '3scale.localhost')
 
-  SphinxIndexationWorker.stubs(:perform_later)
```

Comment:
Now disabling and enabling indexation workers is done in `support/sphinx.rb`. I guess these were needed before to avoid these workers being run later and failing. Presently this should be redundant though.